.github/workflows/build.yml

```diff
@@ -75,48 +88,55 @@ jobs:
           uses: actions/download-artifact@v3
           with:
             name: data-api-jar
-        - uses: aws-actions/configure-aws-credentials@v2
+
+        - name: Configure aws credentials
+          uses: aws-actions/configure-aws-credentials@v2
           with:
-            role-to-assume: ${{ secrets.ECR_ROLE_TO_ASSUME }}
-            aws-region: ${{ vars.ECR_REGION }}
-        # Login to container repo
-        - uses: aws-actions/amazon-ecr-login@v1
+            role-to-assume: ${{ secrets.DEV_ECR_ROLE_TO_ASSUME }}
+            aws-region: ${{ vars.DEV_ECR_REGION }}
+
+        - name: Login ecr
+          uses: aws-actions/amazon-ecr-login@v1
           id: login-ecr
+
         # Build and push docker image to container repo
-        - run: |
-            docker build -t $REGISTRY/$REPOSITORY:$IMAGE_TAG .
-            docker push $REGISTRY/$REPOSITORY:$IMAGE_TAG
+        - name: Build and push image
+          run: |
+            docker build -t $REGISTRY/$REPOSITORY:${{ env.IMAGE_TAG }} .
+            docker push $REGISTRY/$REPOSITORY:${{ env.IMAGE_TAG }}
           env:
             REGISTRY: ${{ steps.login-ecr.outputs.registry }}
-            REPOSITORY: ${{ vars.ECR_REPOSITORY }}
-            IMAGE_TAG: ${{ github.sha }}
+            REPOSITORY: ${{ vars.DEV_ECR_REPOSITORY }}
+            IMAGE_TAG: ${{ needs.define-image-tag.outputs.tag_name }}
+
         - name: delete artifact
           uses: geekyeggo/delete-artifact@v2
           with:
             name: data-api-jar
 
```

Comment:
will need a separate job for this when not on main to delete the artifact.  Either that, or we separate it out

Reply:
resolved